pkgs/applications/backup/vorta/default.nix

```diff
@@ -16,19 +17,31 @@ python3.pkgs.buildPythonApplication rec {
     sha256 = "sha256-AH9CvoWAaNLPLC+NL+VXtYq9CNCMt0AFtC1ZGXSlcHQ=";
   };
 
-  postPatch = ''
-    sed -i -e '/setuptools_git/d' -e '/pytest-runner/d' setup.cfg
-  '';
-
   nativeBuildInputs = [ wrapQtAppsHook ];
 
-  propagatedBuildInputs = with python3.pkgs; [
-    paramiko peewee pyqt5 python-dateutil APScheduler psutil qdarkstyle
-    secretstorage appdirs setuptools
+  propagatedBuildInputs = with python3Packages; [
+    paramiko
+    peewee
+    pyqt5
+    python-dateutil
+    APScheduler
+    psutil
+    qdarkstyle
+    secretstorage
+    appdirs
+    setuptools
   ];
 
-  # QT setup in tests broken.
-  doCheck = false;
+  postPatch = ''
+    sed -i -e '/setuptools_git/d' -e '/pytest-runner/d' setup.cfg
```

Comment:
Please use substituteInPlace and add the ".

Reply:
`substituteInPlace` after `sed` is a no-op here.

Reply:
i forgot to remove the sed